spyderlib/spyder.py

```diff
@@ -2735,6 +2754,11 @@ def restart(self):
         env['SPYDER_ARGS'] = spyder_args
         env['SPYDER_PID'] = str(pid)
         env['SPYDER_IS_BOOTSTRAP'] = str(is_bootstrap)
+        env['SPYDER_RESET'] = str(reset)
+        if os.name == 'nt':
+            env['PYTHONPATH'] = ';'.join(sys.path)
+        else:
+            env['PYTHONPATH'] = ':'.join(sys.path)

```

Comment:
Yep, I would say that is correct. If not running from bootstrap then it must be running from an installed version (site-packages) so the imports should be fine then.

I can make add an if so that the path is updated only if running from bootstrap?